docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -354,3 +354,16 @@ it as the value for the column. If not set, the server will return an error.
 This corresponds to {es-sql}'s request parameter `field_multi_value_leniency`
 (see <<sql-rest-fields>>).
 
+
+`AutoEscapePVA` (default: `true`)::
+The pattern-value arguments make use of `_` and `%` as special characters to
+build patern matching values. Some applications however use these chars as
+regular ones, which can lead to {es-sql} returning more data than the app
+intended. With the auto escaping, the driver will inspect the arguments and
+will escape these special characters if not already done by the application.
+
+
+`IndexIncludeFrozen` (default: `false`)::
+If this parameter is `true`, the server will allow quering of frozen indices.
```

Comment:
Typo: I believe the word is `querying`.

Reply:
Also, I would reformulate a bit to emphasize on the idea of frozen indices being "included" in the search (in addition to the usual normal indices).

```
If this paramter is `true`, the server will include frozen indices in the query execution.
```

Reply:
Thanks, rephrasing applied.